### PR TITLE
October 21st Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by robertengaxway — APPROVED** [review]
approved

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+There are no enhancements or bug fixes in this release.
+
+### Asset Catalog
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Asset Catalog
+
+* **Auto release management** - A new step in the asset creation/edition allows the user to select how the asset can automatically benefits the changes on the associated API Service. By default this feature is not enabled.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:132]: can automatically benefit.

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+There are no known limitations in this release.
+
+### Product Foundry
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+* **Auto release management** - A new step in the product creation/edition allows the user to select how the product can automatically benefits the changes on the associated Asset. By default this feature is not enabled.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:148]: benefit

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+* **Export subscriber contact information** - provider can see all the subscription associated to the product he manage and copy the subscriber email address.  Refer to Product details / Sbscriptions tab.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:149]: a catalog manager can see all subscriptions to the product  and copy the subscriber email address. In addition, they can choose to export all subscriptions, including subscriber contact details to a CSV file.

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+* **Subscription approver role** - a new team role is available to approve the subscription and access request when they are set to be manually approved. This approver role can approve only the one that are coming from product his team own.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:150]: Replace entire paragraph with: a new team role has been introduced that gives users the permission to approve and manage subscriptions and access requests submitted to the product or assets owned by their team.

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+
+There are no known limitations in this release.
+
+### Business insights
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Business insights
+
+* **text** - text.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:166]: There are no feature updates in this release.

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+
+## Consumer experience
+
+### Marketplace
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Marketplace
+
+* **User experience improvement** - The Marketplace home page and browse product page are adpating to the screen size.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:184]: The Marketplace Homepage and the browse product screen have been improved to adapt to different screen sizes.

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+* **Consumer Organization Management** - The provider can decide to allow the consumer to signup to the marketplace and let them manage their own users/teams organization.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:185]: Consumers can now self-register with a Consumer Organization account into the Marketplace and manage their own profile. Platform administrators can configure in the Marketplace Settings if Sign Up is allowed/disallowed.

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+
+#### Bug fixes for Marketplace
+
+* **Wrong owner of the subscription** - The Marketplace subscriptions table displays org name of logged in user not of subscription owner.
+* **Category name are truncated** - The category on prodyct details are now displayed in a single column allowing to see longer name.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:190]: The categories are not displayed in a single column on  the product details screen, allowing users to see the entire category name.

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+* **Incorrect warning displayed** - A warning was wrongly shown in Marketplace when creating a subscription when the owner is not selected yet.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:191]: An inaccurate warning was displayed in the Marketplace when a user created a subscription with no owner selected.

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+* **Error message when subscribing is not clear** - The error essage displayed when the user is not allowed to create subscription with paid plan.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:192]: Unclear error message was displayed when a user was not allowed to subscribe to a paid plan.

File: content/en/docs/amplify_relnotes/20221021_amplify.md [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+
+#### Known issues for Marketplace
+
+* **Access request not visbile from product resources** - From the product / resource details screen, the consumer cannot see the access reqest parameters. The access request are visible from the Application page details.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20221021_amplify.md:196]: Access requests are not accessible from the Product Details -> Resource screen. However, they are visible from the Application details page.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Pull request merged**